src/test/scala/GuessWhoGameSpec/GameSpec.scala

```diff
@@ -1,5 +1,6 @@
 package GuessWhoGameSpec
+import org.scalatest.flatspec.AnyFlatSpec
 
-object GameSpec {
+object GameSpec extends AnyFlatSpec {
 
```

Comment:
Are there tests that need to be added here or should the file be deleted?

Reply:
Done